server/utils/nunjucksSetup.test.ts

```diff
@@ -124,14 +124,14 @@ describe('Nunjucks Filters', () => {
 
   describe('displayAge', () => {
     beforeAll(() => {
-      const fakeDate = new Date('2020-12-14')
+      const fakeDate = new Date('2020-12-14T12:00:00')
```

Comment:
I found setting the time to actually during the day fixed some of the incorrect comparisons - I think perhaps otherwise it defaults to midnight and it doesn't give the right answer on that boundary